traffic_router/core/src/test/resources/publish/V14/CrConfig2.json

```diff
@@ -1236,6 +1235,7 @@
     }
   },
   "config": {
+	"deliveryservice.snapshots": "true",
```

Comment:
this line and the same one below are indented differently than the rest of the file

Reply:
done